src/coreclr/vm/dllimport.h

```diff
@@ -404,7 +406,9 @@ struct PInvokeStaticSigInfo
 private:
     Module* m_pModule;
     Signature m_sig;
+    LPCUTF8 m_entryPointName;
```

Comment:
It does not feel right to add the unmanaged entrypoint name to `PInvokeStaticSigInfo`.

Reply:
I agree that it does seem odd, but the spirit of this class is far more similar to the CrossGen2 data type - `PInvokeMetadata`. Let me see how I can refactor this PR though.